calico-cloud/threat/container-threat-detection.mdx

```diff
@@ -90,6 +91,29 @@ If a malicious or suspicious program is run within the cluster, it will be repor
 
 ![malware-alert-example](/img/calico-cloud/malware-alert-example.png)
 
+### Create an exception for suspicious programs
```

Comment:
### Exclude a process from Security Events alerts

I think this title is a bit clearer. Change the link above as well, please!